Dynamix/MainActivity.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Android.App;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+
+namespace Dynamix
+{
+	[Activity (Label = "Dynamix", MainLauncher = true)]
+	public class MainActivity : Activity
+	{
+		private TextView textField;
+		private Button theButton;
+
+		protected override void OnCreate (Bundle savedInstanceState)
+		{
+			base.OnCreate (savedInstanceState);
+
+			SetContentView (Resource.Layout.activity_main);
+
+			textField = FindViewById<TextView> (Resource.Id.textField);
+			theButton = FindViewById<Button> (Resource.Id.theButton);
+
+			theButton.Click += OnButtonClick;
+		}
+
+		private void OnButtonClick (object sender, EventArgs eventArgs)
+		{
+			dynamic calculatorConstructor = new Java.Interop.Dynamic.DynamicJavaClass ("com/xamarin/dynamix/Calculator");
+
+			var calculator = calculatorConstructor ();
+
+			// normal
+			var answer = calculator.add (3, 6);
+			textField.Text += $"\nNORMAL: 3 + 6 = {answer}";
+
+			// use a callback
+			var listener = new MyListener (textField);
+			calculator.add (2, 4, (IAnswerListener) listener);
+		}
+
+		// this is to create the interface on the Java side
+		[Register ("com/xamarin/dynamix/AnswerListener")]
```

Comment:
This should need a `DoNotGenerateAcw=true` as well, shouldn't it?  Otherwise we'll *generate* this type...

Reply:
I'm wrong; we don't generate interface types.

(Perhaps we *should* -- I remember a feature request to do that, but can't find it -- but we currently *don't*.)

Meaning the lack of `DoNotGenerateAcw=true` isn't a problem *now*, but it *could* be in the future...